src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/service/BookingFacade.kt

```diff
@@ -152,7 +152,7 @@ class BookingFacade(
     val appointment = booking.appointments().single()
     val prison = prisonRepository.findByCode(prisoner.prisonCode)!!
     val contacts = contactsService.getBookingContacts(booking.videoBookingId, user).withAnEmailAddress()
-    val location = locationsInsidePrisonClient.getLocationByKey(appointment.prisonLocKey)!!
+    val location = locationsInsidePrisonClient.getLocationById(appointment.prisonLocationId)
```

Comment:
Not saying make this change, just my thoughts ...

This does make me wonder if we could have done the clients a little different/better???

e.g. `locationsClient.getLocationFor(appointment)`, then let the client determine which field it will use to look up an appointment ...